src/OpenTelemetry.Instrumentation.GrpcNetClient/CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Unreleased
 
+* **Breaking Change** :
+  [SuppressDownstreamInstrumentation](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Instrumentation.GrpcNetClient#suppressdownstreaminstrumentation)
+  option will no longer be supported when used with certain versions of
```

Comment:
Not for this PR but we should write the exact version here when we do the release.